app/logic/validator_score_v1_logic.rb

```diff
@@ -97,29 +93,29 @@ def block_vote_history_get
 
       p.payload[:validators].each do |validator|
         # Get the last root & vote for this validator
-        vh = validator_histories.find { |vh| vh.account == validator.account }
-        vote_h = vote_histories.find { |vote_h| vote_h.vote_account.validator_id == validator.id }
-        if vote_h
-          validator.score.skipped_vote_history_push(vote_h.skipped_vote_percent)
-          skipped_vote_all.push(((best_skipped_vote - vote_h.skipped_vote_percent) * 100).round(2))
-          validator.score.skipped_vote_percent_moving_average_history_push(vote_h.skipped_vote_percent_moving_average)
+        vhs = validator_histories.find { |vh| vh.account == validator.account }
```

Comment:
Why `vhs` instead of `vh`? We get only one here, I'd prefer to name it `validator_history`, and similar below, `vote_hs` to `vote_histories` etc.